src/hotspot/cpu/x86/c1_FrameMap_x86.cpp

```diff
@@ -43,12 +43,8 @@ LIR_Opr FrameMap::map_to_opr(BasicType type, VMRegPair* reg, bool) {
     Register reg = r_1->as_Register();
     if (r_2->is_Register() && (type == T_LONG || type == T_DOUBLE)) {
       Register reg2 = r_2->as_Register();
```

Comment:
FTR `reg2` is unused. (Moreover, `r_2` and `r_2->is_Register()` are redundant on x64.)

Reply:
Right. Cleaned those up too.